specification/confidentialledger/data-plane/readme.csharp.md

```diff
@@ -5,11 +5,11 @@ Please also specify `--csharp-sdks-folder=<path to "SDKs" directory of your azur
 
 ```yaml $(csharp)
 csharp:
-  azure-arm: true
```

Comment:
The autorest configuration for DotNet data plane libraries resides in the azure-sdk-for-net repo. For this service in particular the configuration file is:

https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/confidentialledger/Azure.Security.ConfidentialLedger/src/autorest.md

There is no need for a `readme.csharp.md` file here and it can only cause confusion.  Please remove it or explain why you think it is needed.